Doc/c-api/unicode.rst

```diff
@@ -531,6 +533,11 @@ APIs:
 
    Return the length of the Unicode object, in code points.
 
+   This function checks that *unicode* is a Unicode object, in contrast to
+   :c:func:`PyUnicode_GET_LENGTH`, which performs no error checking.
```

Comment:
This isn't using imperative mood. But then again `PyUnicode_ReadChar` doesn't do so for the same text either.

Actually quite a few of the existing documentation doesn't use the imperative mood. Some examples in the return statements are:
- PyUnicode_AsWideCharString
- PyUnicode_Compare
- PyUnicode_READY
- PyUnicode_FromEncodedObject
- PyUnicode_CopyCharacters
- PyUnicode_AsUCS4

Reply:
Now updated this as well. The existing cases continue to exist.